docs/data-cookbooks/run-tasks-in-lambda-or-docker.md

```diff
@@ -90,7 +90,6 @@ Given it has been discovered this task can no longer run in AWS Lambda, you can
 ```hcl
 resource "aws_sfn_activity" "queue_granules" {
   name = "${var.prefix}-QueueGranules"
-  tags = local.default_tags
```

Comment:
I assume we aren't including these in the docs since `local.default_tags` is gone now. Is there value in showing use of `var.tags` so people's resources would be tagged?

Reply:
I'd say not in this case, since it isn't really related to what this doc is trying to show.